client/devdocs/design/index.jsx

```diff
@@ -83,44 +97,44 @@ export default React.createClass( {
 					</SearchCard>
 				}
 				<Collection component={ this.props.component } filter={ this.state.filter }>
-					<Accordions />
```

Comment:
`React.cloneElement` takes care of shallow merging the new props with the old ones and it preserves things like `key` and `ref`.